plugins/docker/docker_

```diff
@@ -54,35 +55,60 @@ from multiprocessing import Process, Queue
 
 
 def print_containers_status(client):
-    running = 0
-    paused = 0
-    created = 0
-    restarting = 0
-    removing = 0
-    exited = 0
-    dead = 0
-    for container in client.containers.list():
+    running = []
+    paused = []
+    created = []
+    restarting = []
+    removing = []
+    exited = []
+    dead = []
+    for container in client.containers.list(all=True):
         if container.status == 'running':
-            running += 1
+            running.append(container.name)
         elif container.status == 'paused':
-            paused += 1
+            paused.append(container.name)
         elif container.status == 'created':
-            created += 1
+            created.append(container.name)
         elif container.status == 'restarting':
-            restarting += 1
+            restarting.append(container.name)
         elif container.status == 'removing':
-            removing += 1
+            removing.append(container.name)
         elif container.status == 'exited':
-            exited += 1
+            exited.append(container.name)
         elif container.status == 'dead':
-            dead += 1
-    print('running.value', running)
-    print('paused.value', paused)
-    print('created.value', created)
-    print('restarting.value', restarting)
-    print('removing.value', removing)
-    print('exited.value', exited)
-    print('dead.value', dead)
+            dead.append(container.name)
+    print('running.value', len(running))
+    print('running.extinfo', ', '.join(running))
+    print('paused.value', len(paused))
+    print('paused.extinfo', ', '.join(paused))
+    print('created.value', len(created))
+    print('created.extinfo', ', '.join(created))
+    print('restarting.value', len(restarting))
+    print('restarting.extinfo', ', '.join(restarting))
+    print('removing.value', len(removing))
+    print('removing.extinfo', ', '.join(removing))
+    print('exited.value', len(exited))
+    print('exited.extinfo', ', '.join(exited))
+    print('dead.value', len(dead))
+    print('dead.extinfo', ', '.join(dead))
+
+def str_image(image):
+    return f"{image.short_id} ({round(image.attrs['Size']/1024**2, 2)} MiB)"
+
+def print_images_count(client):
+    images = client.images.list()
+    intermediate = [ i for i in client.images.list(all=True)
+                     if i not in images ]
+    dangling = client.images.list(filters={'dangling':True})
+    images = [ i for i in images
+               if i not in dangling ]
+
+    print('intermediate_quantity.value', len(intermediate))
+    print('intermediate_quantity.extinfo', ', '.join([str_image(i) for i in intermediate]))
```

Comment:
Style nitpick: you can remove `[...]` from the list comprehension and thus turning it into a generator expression. Again, just a matter of taste ...